test/e2e/spec/api_spec.js

```diff
@@ -35,6 +35,8 @@ describe("API Versioning", function () {
 describe("API v1", function () {
     this.timeout(3000);
 
+    let publicSuffix = 'dedyn.io';
```

Comment:
remove static dependency on `'dedyn.io'`

Reply:
nevermind, everything in this function depends on that name. Also, it's going to be replaced anyway

Reply:
not part of this PR. marked as TODO in e0131c7